### PR TITLE
Second level menu

### PR DESCRIPTION
**Pull request by wlatanowicz**

### CODE REVIEW

File: src/lib/Menu.js [hunk 1]
```diff
@@ -13,26 +13,111 @@ const NavMenu = styled.ul`
...
 
 const Nav = styled.header`
   background-color: white;
   box-shadow: ${palette.MENU_BOX_SHADOW};
   border-radius: 5px;
-  padding: 24px 40px;
+  padding: 0px 40px;
 `;
 
+class Submenu extends Component {
```
Comment (feimosi) [inline src/lib/Menu.js]: Could be a functional component.

File: src/lib/Menu.js [hunk 2]
```diff
@@ -13,26 +13,111 @@ const NavMenu = styled.ul`
...
+  render() {
+    return (
+      <NavMenu>
+        { this.props.links.map((item, index) =>
+          <SubNavItem key={index}>
```
Comment (feimosi) [inline src/lib/Menu.js]: `key`

File: src/lib/Menu.js [hunk 3]
```diff
@@ -13,26 +13,111 @@ const NavMenu = styled.ul`
...
+}
+
 export class Menu extends Component {
   render() {
     return (
       <Nav>
         <NavMenu>
           { this.props.links.map((item, index) =>
-            <NavItem key={ index }>
+            <NavItem key={item.name + index} className={item.submenu ? 'with_submenu' : ''}>
```
Comment (feimosi) [inline src/lib/Menu.js]: We shouldn't really mix `className` with styled-components. My first guess would be to use two separate components:
`NavItem` and ``NavItem.extend`...` `` with conditional rendering.

Comment (wlatanowicz) [reply src/lib/Menu.js]: Done

File: src/lib/Menu.js (hunk 2 above)
Comment (wlatanowicz) [reply src/lib/Menu.js]: OK

File: src/lib/Menu.js (hunk 1 above)
Comment (wlatanowicz) [reply src/lib/Menu.js]: OK

**Pull request merged**